### PR TITLE
[RETINA-336] Track tool nfc tag status

### PR DESCRIPTION
**Pull request by Jbcampbe**
Depends on https://github.com/Renascentinc/retina-api/pull/134

- When a tool is created its default status is `tagged: false`
- When a tag is successfully encoded the status is automatically updated to `tagged: true`
- Manually changing the tools `tagged` status can be done from the tool info page.

screen shot 2019-03-01 at 8 31 29 pm
screen shot 2019-03-01 at 8 31 44 pm
screen shot 2019-03-01 at 10 03 52 pm

### CODE REVIEW

**Comment by ekraihan** [comment]
I wondered, on the tool history page about having the field Tagged say something like "Yes" and "No" versus booleans. So like

Tagged: Yes -> No

Just an idea.

**Comment by ekraihan** [comment]
Are there any restrictions around a user (vs an admin) changing tagged status? Maybe a user would just see the Yes/No value on the tool info page instead of a slider?

**Comment by Jbcampbe** [comment]
> Are there any restrictions around a user (vs an admin) changing tagged status? Maybe a user would just see the Yes/No value on the tool info page instead of a slider?

I think standard users will be allowed to tag tools so I would guess we wouldn't want to restrict this.

NOTE: the toggle is disabled unless you edit the tool. Only then can you flip it and save so it should make accidents a lot harder

File: src/components/tool-search-input.vue [hunk 1]
```diff
@@ -135,7 +135,26 @@ export default {
         }
       ]
 
-      let items = statuses.concat(this.searchableConfigItems).concat(this.locations)
+      let nfcTagStatus = [
+        {
+          name: 'Has NFC Tag',
+          text: 'Has NFC Tag',
```
Comment (ekraihan) [inline src/components/tool-search-input.vue]: When I was testing the search feature for the tagged attribute, my first attempt was to search "tagged" which brought up no auto complete options (I had to type "Has" etc.). I wondered about making these "Tagged" and "Not Tagged"?

**Comment by ekraihan** [comment]
> > Are there any restrictions around a user (vs an admin) changing tagged status? Maybe a user would just see the Yes/No value on the tool info page instead of a slider?
> 
> Standard users are able to tag tools so I would think we should allow them to edit its status (for example if a tag falls off while it is in their care)
> 
> NOTE: the toggle is disabled unless you edit the tool. Only then can you flip it and save so it should make accidents a lot harder

Oh, except users can't edit a tool 😄

**Pull request closed without merging**